docs/building-a-checkout-path.md

```diff
@@ -0,0 +1,73 @@
+
+If your client does not expect to be handling multiple fulfillment groups, or if you want the same selection to be used on every fulfillment group, you may choose to show only one selection list. But you’ll still need to set that selection on each group individually for order creation to succeed.
+
+### Example Storefront Component Library components
+- The `SelectableList` React component shows a radio-like list of options, allowing you to select just one. This is a good way to show the list of available fulfillment types.
+- You can use the `FulfillmentOptionsCheckoutAction` component with an action supplied to the `CheckoutActions` component to collect a fulfillment option selection as part of a step-by-step checkout flow. This wraps `SelectableList` and takes care of some of the complexity for you, versus using `SelectableList` directly.
+
+### GraphQL
+Call the `updateFulfillmentOptionsForGroup` mutation as necessary during checkout to update the `availableFulfillmentOptions` property for all fulfillment groups of a cart with fresh price quotes. These need to be recalculated every time the items in that group change. When the order is placed, the chosen option for each group will have its prices recalculated one last time. If the prices do not match, order creation will fail.
+
+Use the `selectFulfillmentOptionForGroup` mutation if you have a cart and want to store the selected fulfillment option on each of the fulfillment groups on that cart. This is recommended for data completeness and to avoid re-entry if the checkout flow is interrupted. However, you could also just cache this data on the client until you call the place order mutation.
+
+> NOTE: Call this mutation once for each fulfillment group.
+
+## Obtain payment information
+Check to see which payment method is the default for the shop, and show the checkout component provided by that payment method. If you know that the shop for the client you are building will only ever have a single payment method, you can skip the check and always show that component. If there could be multiple payment methods enabled (for example, credit card, PayPal, and Apple Pay), then you should display a list and allow the shopper to choose one.
+
+Ultimately, the payment method component should give you some kind of data, usually a reference ID or token, which you can pass to the server with the order details to create the order. If the payment details are invalid, the order will not be created.
+
+### Example Storefront Library components
+- The [StripeCard](https://github.com/reactioncommerce/example-storefront/tree/trunk/components/StripeCard) component collects credit card information from a shopper and gives you a Stripe token for it. This is a very secure, PCI-compliant way of providing credit card payment, but it will only work if you [sign up for a Stripe account](https://dashboard.stripe.com/register).
+
+
+### GraphQL
```

Comment:
Seems like this should go before "obtain payment information"?

Reply:
It's an h3, so it's part of the enclosing h2 "obtain payment information"